File: 백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.cs
using System.Text;

int num = int.Parse(Console.ReadLine());
int[] arr = new int[num];
for (int i = 1; i <= num; i++)
{
    arr[i - 1] = int.Parse(Console.ReadLine());
}
Array.Sort(arr);
StringBuilder sb = new StringBuilder();
for(int i = 0; i < num; i++)
{
    sb.Append(arr[i] + "\n");
}
Console.WriteLine(sb.ToString());

File: 백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.Arrays;
public class Main {
    public static void main(String[] args)throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        int num = Integer.parseInt(br.readLine());
        Integer[] nums = new Integer[num];
        
        for(int i = 0; i < num; i++){
            nums[i] = Integer.parseInt(br.readLine());
        }
        Arrays.sort(nums);
        
        for(int i = 0; i < num; i++){
            bw.write(nums[i]+"\n");
        }
        bw.flush();
        bw.close();
    }
}